anta/tests/interfaces.py

```diff
@@ -883,3 +883,134 @@ def test(self) -> None:
                         output["speed"] = f"{custom_division(output['speed'], BPS_GBPS_CONVERSIONS)}Gbps"
                 failed_log = get_failed_logs(expected_interface_output, actual_interface_output)
                 self.result.is_failure(f"For interface {intf}:{failed_log}\n")
+
+
+class VerifyLACPInterfacesStatus(AntaTest):
+    """Verifies the Link Aggregation Control Protocol(LACP) status for interfaces.
+
+    - Verify that the interface is correctly added to LAG (Bundled in PortChannel).
+    - Ensure that the synchronization is established.
+    - Ensure that the interfaces are in the correct state for collecting and distributing the traffic.
+    - Validate that LACP settings, such as timeouts are correctly configured.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the provided interfaces are bundled in port channel and all specified parameters are correct.
+    * Failure: The test will fail if any interface is not bundled in port channel and any of specified parameter is not correct.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.interfaces:
+      - VerifyLACPInterfacesStatus:
+          interfaces:
+            - name: Ethernet1
+              portchannel: Port-Channel100
+    ```
+    """
+
+    name = "VerifyLACPInterfacesStatus"
+    description = "Verifies the Link Aggregation Control Protocol(LACP) status of the provided interfaces."
+    categories: ClassVar[list[str]] = ["interfaces"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaTemplate(template="show lacp interface {interface}", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifyLACPInterfacesStatus test."""
+
+        interfaces: list[Interfaces]
+        """List of interfaces with their expected state."""
+
+        class Interfaces(BaseModel):
```

Comment:
Can you change the class name as it creates confusion for the custom type as `Interface`

Reply:
Updated class name LACPInterfaces